### DISCUSSION TITLE
Ability to select and download multiple files at once

### DISCUSSION THREAD

**Discussion by SinTan1729** [Radiokot/photoprism-android-client#85] [Ideas]
Currently, when I select multiple files, the only option is to share them. It'll be nice if there were an option to download the files as well.

It's possible to bypass this by sharing the files to a file manager and saving them with it (e.g. MiXplorer supports this functionality), but having the function built in will be useful. Since the app is downloading the files before sharing anyway, I think the option should definitely be there.

**Comment by Radiokot** [comment]
Hi. I agree, the ability to download selected files should be added.

**Reply by Radiokot** [reply]
[1.20.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.20.0(30)) release contains this feature.
